app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] This association is well set up. However, when I try to create a post, I get an error message that 'Author must be present', because with this set up Rails expects to find an `author_id` in your table whereas it finds `user_id`. Kindly remove the `user_id` key in the table and ensure `author_id` is there instead.

  ![sadiq-post-user](https://user-images.githubusercontent.com/36542986/162941867-2d027abf-fbbb-434f-bce7-66cc8f433285.jpg)